src/sage/plot/graphics.py

```diff
@@ -2874,6 +2876,11 @@ def matplotlib(self, filename=None,
                 weight=lopts.pop('font_weight', 'medium'),
                 variant=lopts.pop('font_variant', 'normal'))
             color = lopts.pop('back_color', 'white')
+            with contextlib.suppress(KeyError):
```

Comment:
This is a big hammer... why not `if "loc" in lopts`?

Reply:
Sure, I don't know what I was thinking. Done.